src/app/services/litmus.services.ts

```diff
@@ -6,17 +6,9 @@ import { litmusstatus,litmuslog,jobname } from "../model/data.model";
 @Injectable()
 export class LitmusService {
   private apiurl: string;
-  private host: string;
+  
```

Comment:
This blank line is not required

Reply:
i think is should be good if separate the two thing by line.
the first part contains the variable and below that is a constructor  that's why I put the line here

Reply:
okay!!